x-pack/plugins/stack_alerts/server/alert_types/es_query/alert_type.ts

```diff
@@ -119,6 +116,28 @@ export function getAlertType(logger: Logger): RuleType<
     }
   );
 
+  const actionVariableSearchTypeLabel = i18n.translate(
+    'xpack.stackAlerts.esQuery.actionVariableContextSearchTypeLabel',
+    {
+      defaultMessage: 'The type of search is used.',
+    }
+  );
+
+  const actionVariableSearchConfigurationLabel = i18n.translate(
+    'xpack.stackAlerts.esQuery.actionVariableContextSearchConfigurationLabel',
+    {
+      defaultMessage:
+        'Serialized search source fields used to fetch the documents from Elasticsearch.',
+    }
+  );
+
+  const actionVariableContextLinkLabel = i18n.translate(
+    'xpack.stackAlerts.esQuery.actionVariableContextLinkLabel',
+    {
+      defaultMessage: 'A link to see records that triggered this alert.',
+    }
+  );
+
```

Comment:
Should we mention that some variables will only be used with `esQuery` and some of them with `searchSource`?

Reply:
Wording for link context parameter

![E6270BE4-148C-471A-A9BA-84177ECB4FAA_4_5005_c](https://user-images.githubusercontent.com/39378793/155538910-a0d77455-5b62-4265-bdb1-3c030adb16ab.jpeg)

Reply:
So, would current explanation be sufficient for link?

Reply:
Think we should change to a description like this 

`If the alert rule was created in Discover, the link will navigate to Discover showing the records that triggered this alert. In the other case the link will navigate to the rule's status page.`

FYI @gchaps for better wording